ekf/sensors_client.c

```diff
@@ -92,32 +96,33 @@ void sensImu2(sensor_event_t * accel_evt, sensor_event_t * gyro_evt, sensor_even
 	/* decompose sensorhub output */
 	for (int j = 0; j < data->size; ++j) {
 		switch (data->events[j].type) {
-		case SENSOR_TYPE_ACCEL:
-			// printf("Accel, timestamp: %llu\n", data->events[j].timestamp);
-			// printf("acce - x: %u, y: %u, z: %u\n", data->events[j].accels.accelX, data->events[j].accels.accelY, data->events[j].accels.accelZ);
-			*accel_evt = data->events[j];
-			break;
-
-		case SENSOR_TYPE_MAG:
-			// printf("Baro, timestamp: %llu\n", data->events[j].timestamp);
-			// printf("baro - pressure: %u\n", data->events[j].baro.pressure);
-			*mag_evt = data->events[j];
-			break;
-
-		case SENSOR_TYPE_GYRO:
-			// printf("Gyro, timestamp: %llu\n", data->events[j].timestamp);
-			// printf("gyro - x: %u, y: %u, z: %u\n", data->events[j].gyro.gyroX, data->events[j].gyro.gyroY, data->events[j].gyro.gyroZ);
-			*gyro_evt = data->events[j];
-			break;
-
-		default:
-			break;
+			case SENSOR_TYPE_ACCEL:
+				// printf("Accel, timestamp: %llu\n", data->events[j].timestamp);
+				// printf("acce - x: %u, y: %u, z: %u\n", data->events[j].accels.accelX, data->events[j].accels.accelY, data->events[j].accels.accelZ);
```

Comment:
Maybe disable those lines using some define or define a debug macro?
```suggestion
#if EKF_DEBUG
				printf("Accel, timestamp: %llu\n", data->events[j].timestamp);
				printf("acce - x: %u, y: %u, z: %u\n", data->events[j].accels.accelX, data->events[j].accels.accelY, data->events[j].accels.accelZ);
#endif
```